packages/babel-plugin-proposal-decorators/src/transformer-legacy.js

```diff
@@ -38,13 +38,13 @@ function applyEnsureOrdering(path) {
     : path.get("properties")
   ).reduce((acc, prop) => acc.concat(prop.node.decorators || []), []);
 
-  const identDecorators = decorators.filter(
+  const nonIdentDecorators = decorators.filter(
     decorator => !t.isIdentifier(decorator.callee),
   );
-  if (identDecorators.length === 0) return;
+  if (nonIdentDecorators.length === 0) return;
 
   return t.sequenceExpression(
-    identDecorators
+    nonIdentDecorators
```

Comment:
This renaming is not a functional change, but it seemed appropriate because this array contains only decorators whose `callee` properties are _not_ `Identifier`s.